pages/glossary/changes-in-content.md

```diff
@@ -4,15 +4,25 @@ key: changes-in-content
 unambiguous: true
 objective: true
 input_aspects:
+  - Accessibility tree
+  - Audio output
   - CSS styling
   - DOM tree
 ---
 
-Changes in the content of a [web page][] occur when at least one of the following occurs:
+A _change in the content_ of a [web page][] caused by an event occurs when, by comparing the [web page][] before and after the event firing, at least one of the following occurs:
 
-- **visible changes:** Visible changes in content occur when the rendered pixels change in any part of the document that is currently within the viewport or that can be brought into the viewport via scrolling; or
-- **accessibility tree changes:** Changes in the content of the accessibility tree occur when any state, property or event of a node representing an [accessible object][] of the tree changes, or when any node is inserted in, or removed from the tree; or
-- **audible changes:** Audible changes in content occur when the audio rendered by the [web page][] change.
+- **visible changes:** the rendered pixels change in any part of the document that is currently within the [viewport][] or that can be brought into the [viewport][] via scrolling; or
+- **accessibility tree changes:** any state, property or event of a node representing an [accessible object][] of the [accessibility tree][] changes, or any node is inserted in, or removed from the [accessibility tree][]; or
+- **audible changes:** the audio rendered by the [web page][] changes.
```

Comment:
I'm actually not sure whether "rendered" is the correct word to use for audio (or whether it is purely visual).

https://html.spec.whatwg.org/multipage/rendering.html
https://html.spec.whatwg.org/multipage/rendering.html#being-rendered
https://html.spec.whatwg.org/multipage/rendering.html#embedded-content-rendering-rules

This part is non normative, but the second link strongly suggests that we should only use "rendered" for visual stuff, not for audio.

Reply:
The W3C Web Audio API uses "audio rendering" has does almost everyone that works in the field :-)
But I agree that the HTML spec does not, so I'll change to "reproduced"